ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_p2p_links.schema.yml

```diff
@@ -105,10 +105,13 @@ $defs:
           type: bool
           default: false
           description: Enable BFD (only considered for BGP).
-        ptp_enable:
-          type: bool
-          default: false
-          description: Enable PTP
+        ptp:
+          description: PTP parameters.
+          type: dict
+          enabled:
```

Comment:
ok found the issue - here it should be 

```
ptp:
   description: PTP parameters.
   type: dict
   keys:
     enabled:
...
```

missing keys make things not behave properly

Reply:
Fixed